i18n/es.jsonp.js

```diff
@@ -241,17 +242,17 @@ document.localeJson = {
       "about-privacy-short-title": "Versión corta:",
       "about-privacy-short-para1": "Kiwix JS funciona sin conexión a Internet y no recopila ni registra ninguno de los datos personales del usuario. Sólo recuerda su historial de navegación durante la duración de una sesión (con el fin de volver a las páginas vistadas anteriormente). Este historial se pierde al salir de la aplicación y no se registra de ninguna manera.",
       "about-privacy-long-title": "Versión larga:",
-      "about-privacy-long-para1": "Esta aplicación sólo lee los archivos que usted selecciona explícitamente en su dispositivo junto con los archivos del propio paquete del software. Algunos archivos ZIM contienen contenido activo (scripts) que pueden, en circunstancias excepcionales, intentar ponerse en contacto con servidores externos para obtener archivos incidentales como fuentes. Tenemos una Política de Seguridad de Contenido que intenta evitar esto, pero no es posible garantizar que todas estas solicitudes se bloqueen, por ejemplo, si un ZIM contuviera código malicioso intencional que pudiera romper el sandbox. Tenga en cuenta que los scripts sólo se ejecutarán si habilita el <a href=\"#modes\">Modo ServiceWorker</a> (predeterminado) en Configuración. Sin embargo, si cree que su acceso a Internet no es seguro, o está siendo observado o censurado, le recomendamos que cierre completamente su acceso a Internet (datos o WiFi) antes de utilizar la aplicación.",
+      "about-privacy-long-para1": "Esta aplicación sólo lee los archivos que usted selecciona explícitamente en su dispositivo junto con los archivos del propio paquete del software. Algunos archivos ZIM contienen contenido activo (scripts) que pueden, en circunstancias excepcionales, intentar ponerse en contacto con servidores externos para obtener archivos incidentales como fuentes. Tenemos una Política de Seguridad de Contenido que intenta evitar esto, pero no es posible garantizar que todas estas solicitudes se bloqueen, por ejemplo, si un ZIM contuviera código malicioso intencional que pudiera romper el sandbox. Tenga en cuenta que los scripts sólo se ejecutarán si habilita el <a href=\"#modes\">Modo ServiceWorker</a> (predeterminado) en Configuración. Sin embargo, si cree que su acceso a Internet no es Seguro, o está siendo observado o censurado, le recomendamos que cierre completamente su acceso a Internet (datos o WiFi) antes de utilizar la aplicación.",
```

Comment:
Again, "seguro" here has notning to do with the mode, and so should not be capitalized.